packages/odyssey-react-mui/src/DataTable/components/DataTableRowActions.tsx

```diff
@@ -0,0 +1,120 @@
+/*!
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { MRT_Row, MRT_RowData } from "material-react-table";
+import { Fragment, ReactElement, memo } from "react";
+import { Button } from "../../Button";
+import { MenuItem } from "../../MenuItem";
+import { Box } from "@mui/material";
+import { MenuButton } from "../../MenuButton";
+import {
+  ArrowBottomIcon,
+  ArrowDownIcon,
+  ArrowTopIcon,
+  ArrowUpIcon,
+  MoreIcon,
+} from "../../icons.generated";
+import { DataTableProps } from "../DataTable";
+
+export type DataTableRowActionsProps = {
+  row: MRT_Row<MRT_RowData>;
+  rowIndex: number;
+  rowActionButtons?: (
+    row: MRT_RowData
+  ) => ReactElement<typeof Button | typeof Fragment>;
+  rowActionMenuItems?: (
+    row: MRT_RowData
+  ) => ReactElement<typeof MenuItem | typeof Fragment>;
+  totalRows?: DataTableProps["totalRows"];
+  updateRowOrder?: ({
+    rowId,
+    newRowIndex,
+  }: {
+    rowId: string;
+    newRowIndex: number;
+  }) => void;
+};
+
+const DataTableRowActions = ({
+  row,
+  rowIndex,
+  rowActionButtons,
+  rowActionMenuItems,
+  totalRows,
+  updateRowOrder,
+}: DataTableRowActionsProps) => (
+  <Box display="flex">
+    {rowActionButtons?.(row)}
+    {(rowActionMenuItems || updateRowOrder) && (
+      <MenuButton
+        endIcon={<MoreIcon />}
```

Comment:
Is there any possibility consumers could influence the icon used? Also, any possibility consumers could add a label to this menu button (to have text + icon) ?

Reply:
We aren't planning this right now, for the sake of making every table completely consistent, but I can add this to the list of things to consider as fast-follows!